docs/basic_analysis.rst

```diff
@@ -5,6 +5,8 @@ Basic Analysis
 The specutils package comes with a few basic spectral analytic functions.
 More extensive and involve analysis techniques will be available in another
 package, `specreduce <https://github.com/astropy/specreduce>`_.
+We also recommend that you consider the astropy affiliated package
```

Comment:
I think this could use a bit more context about *who* would want this.  Maybe something like "for more advanced tools for line-fitting of UV/optical/IR absorption systems, you may want to look at linetools"?

Reply:
Given that this is routinely used by unskilled undergraduates, it would
be wrong to refer to the tools as 'advanced'.  And I'm not sure we have
any line-fitting of interest.

Reply:
have taken another stab.